src/pushsource/_impl/model/azure.py

```diff
@@ -65,3 +65,31 @@ def __attrs_post_init__(self):
                 'The attribute "legacy_sku_id" must only be set when'
                 ' "support_legacy" is True.'
             )
+
+    @classmethod
+    def _from_data(cls, data):
+        """Instantiate AmiPushItem from raw list or dict"""
```

Comment:
The doc string is false, referring to the wrong class.

It's reasonable to not even bother with a comment / doc string like this, where it's a private method and the purpose of the method is easy to figure out anyway.  A comment which doesn't exist can't be wrong.